Starforge/Editor/Tools/TileRectangleTool.cs

```diff
@@ -1,92 +1,75 @@
 ﻿using Microsoft.Xna.Framework;
-using Microsoft.Xna.Framework.Input;
 using Starforge.Core;
-using Starforge.Core.Input;
 using Starforge.Editor.Actions;
-using Starforge.Editor.UI;
-using Starforge.MapStructure;
+using Starforge.Map;
+using Starforge.Mod.Content;
 
 namespace Starforge.Editor.Tools {
-
     public class TileRectangleTool : Tool {
+        private Rectangle Hint = new Rectangle(-8, -8, 8, 8);
```

Comment:
Tool locations are set OoB because they don't update their position when you first select them, causing them to just appear and stay in the corner until you move your mouse from the tool menu. Will add a comment